.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: node_js
-cache: yarn
+cache:  yarn
 node_js:
 - "8"
 install:
-- yarn install
-script: gitbook install && gitbook build
+  - yarn install
+script:
+  - markdownlint . --ignore node_modules
+  - gitbook install && gitbook build
```

Comment:
Looking at the logs Travis is not behaving as I expected, it doesn't fail fast even though the linting fails.

A pattern I think is nice is using Makefiles, that would give us some freedom to say "Only run X if Y succeeded" without having to write bash in this file. Then in this file we could just say "make ci" for script.

The other alternative would be to do `markdownlint . --ignore node_modules && gitbook install && gitbook build` but then I'd much prefer to just put it under make target.

See some discussions [here](https://github.com/travis-ci/travis-ci/issues/1066).

(Also gitbook install + build takes 60s and linting only 2s so having linting first makes sense)

Reply:
This change could be out of scope of this PR though, it's up to you.

Reply:
One option from https://github.com/travis-ci/travis-ci/issues/1066: 

```yaml
  - set -e
  - command1
  - command2
  - set +e
```

Another is to have two build jobs, one for linting, one for build. But as you say, linting is so quick it's probably not worth the overhead of setting up the build job.

I'll do `command1 && command2` for this PR.